force-app/main/default/lwc/reviewSessions/reviewSessions.js

```diff
@@ -0,0 +1,185 @@
+                label: this.sessionDateLabel,
+                fieldName: SESSION_DATE_FIELD.fieldApiName,
+                hideDefaultActions: true,
+            },
+            {
+                label: this.labels.startTime,
+                fieldName: STARTTIMEAPINAME,
+                hideDefaultActions: true,
+            },
+            {
+                label: this.labels.endTime,
+                fieldName: ENDTIMEAPINAME,
+                hideDefaultActions: true,
+            },
+            {
+                label: "",
+                fieldName: "",
+                type: "button-icon",
+                hideDefaultActions: true,
+                typeAttributes: {
+                    iconName: "utility:clear",
+                    variant: "bare",
+                    iconPosition: "left",
+                    disabled: false,
+                },
+            },
+        ];
+
+        this.columns = COLUMNS;
+        this.setupDataTableData();
+    }
+
+    setupDataTableData() {
+        const FIRSTSESSIONNAME = "Family Class (Thu-Fri) Thursday: 7/6/2020";
+        const SECONDSESSIONNAME = "Family Class (Thu-Fri) Thursday: 7/7/2020";
+        const THIRDSESSIONNAME = "Family Class (Thu-Fri) Thursday: 7/13/2020";
+        const FOURTHSESSIONNAME = "Family Class (Thu-Fri) Thursday: 7/14/2020";
+        const FIFTHSESSIONNAME = "Family Class (Thu-Fri) Thursday: 7/20/2020";
+        const SIXTHSESSIONNAME = "Family Class (Thu-Fri) Thursday: 7/21/2020";
+
+        const DATA = [
+            {
+                id: "1",
+                Name: FIRSTSESSIONNAME,
+                SessionDate__c: "7/6/2020",
+                startTime: "1:00 PM",
+                endTime: "3:00 PM",
+            },
+            {
+                id: "2",
+                Name: SECONDSESSIONNAME,
+                SessionDate__c: "7/7/2020",
+                startTime: "1:00 PM",
+                endTime: "3:00 PM",
+            },
+            {
+                id: "3",
+                Name: THIRDSESSIONNAME,
+                SessionDate__c: "7/13/2020",
+                startTime: "1:00 PM",
+                endTime: "3:00 PM",
+            },
+            {
+                id: "4",
+                Name: FOURTHSESSIONNAME,
+                SessionDate__c: "7/14/2020",
+                startTime: "1:00 PM",
+                endTime: "3:00 PM",
+            },
+            {
+                id: "5",
+                Name: FIFTHSESSIONNAME,
+                SessionDate__c: "7/20/2020",
+                startTime: "1:00 PM",
+                endTime: "3:00 PM",
+            },
+            {
+                id: "6",
+                Name: SIXTHSESSIONNAME,
+                SessionDate__c: "7/21/2020",
+                startTime: "1:00 PM",
+                endTime: "3:00 PM",
+            },
+        ];
+        this.data = DATA;
+    }
+
+    handleAddRows() {
+        this.setupDataTableData();
+        this.data.push({
+            id: "",
```

Comment:
When adding a row to the table we will need to create a unique id here. We had a method in the util for Vol-Man that we got from DSO originally: 
`const createUUID = () => {
    let currentTime = new Date().getTime();
    let uuid = "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx".replace(/[xy]/g, function(char) {
        let randomNumber = (currentTime + Math.random() * 16) % 16 | 0;
        currentTime = Math.floor(currentTime / 16);
        return (char === "x" ? randomNumber : (randomNumber & 0x3) | 0x8).toString(16);
    });
    return uuid;
};`

Reply:
When I delete one or more rows and then click the Add button, the rows are coming back.